doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -64,7 +64,13 @@ Bluetooth LE
 Bluetooth mesh
 --------------
 
-|no_changes_yet_note|
+* Fixed:
```

Comment:
Also wondering if these two changes should rather go in under libraries.

Reply:
As this is a bug-fix for existing features, I think it is correct where it is. It seems, only new feature additions are reported in the library section.

Reply:
Oki, we leave it there